client/src/views/Profile.jsx

```diff
@@ -0,0 +1,351 @@
+import React from "react";
+import styled from "styled-components/macro";
+import Container from "react-bootstrap/Container";
+import Profilephoto from "../assets/ImageLarge.png";
+import { useHistory } from "react-router-dom";
+import axios from "axios";
+import { FaCheckCircle, FaBullseye, FaAngleRight } from "react-icons/fa";
+const Profile = ({ auth }) => {
+  const history = useHistory();
+  const ClickOngoing = (e) => {
+    e.preventDefault();
+    history.push("/goals");
+  };
+
+  const DeleteHandler = (e) => {
+    e.preventDefault();
+    axios
+      .delete("/user", {
+        withCredentials: true,
+      })
+      .then((response) => {
+        alert("Account deleted");
+        history.push("/onboarding");
+      })
+      .catch((err) => {
+        alert("Cannot delete account");
+      });
+  };
+
+  return (
+    <Wrapper>
+      <CustomContainer>
+        <Title>Profile</Title>
+        <InformationWrapper>
+          <BlockDiv>
+            <ProfileImage src={Profilephoto} />
+            <Name>{auth.username}</Name>
+          </BlockDiv>
+          <FlexDiv>
+            <Line />
+            <BlockDiv>
+              <Joined>Joined</Joined>
+              <FlexDiv>
+                <Times>
+                  {Math.floor(
+                    (Date.now() - Date.parse(auth.joinDate)) /
+                      (1000 * 3600 * 24)
+                  )}
+                </Times>
+                <DaysAgo> days ago</DaysAgo>
+              </FlexDiv>
+            </BlockDiv>
+          </FlexDiv>
+        </InformationWrapper>
+
+        <Flex>
+          <ItemWrapper>
+            <CustomContainer1>
+              <ItemDiv>
+                <ItemText>
+                  <FaCheckCircle />
+                </ItemText>
+              </ItemDiv>
+              <Status>{auth.completedGoals.length}</Status>
+              <GoalTitle>Compelted</GoalTitle>
+            </CustomContainer1>
+          </ItemWrapper>
+
+          <SecondItemWrapper>
+            <CustomContainer1>
+              <SecondItemDiv>
+                <ItemText>
+                  <FaBullseye />
+                </ItemText>
+              </SecondItemDiv>
+              <Status>{auth.onGoingGoals.length}</Status>
+              <GoalTitle>Ongoing</GoalTitle>
+            </CustomContainer1>
+          </SecondItemWrapper>
+        </Flex>
+
+        <GoalsDiv onClick={ClickOngoing}>
+          <GoalsTitleDiv>
+            <SecondItemTextDiv>
+              <FaBullseye />
+            </SecondItemTextDiv>
+            <GoalsTitle>Ongoing Goals</GoalsTitle>
+          </GoalsTitleDiv>
+          <Angleright>
+            <FaAngleRight />
+          </Angleright>
+        </GoalsDiv>
+        <QuitButton onClick={DeleteHandler}>Delete Account</QuitButton>
+      </CustomContainer>
+    </Wrapper>
+  );
+};
+
+export default Profile;
+
```

Comment:
I believe the font size is way too big, I will update the design on figma as reference